src/renderers/dom/client/wrappers/ReactDOMSelect.js

```diff
@@ -44,6 +44,17 @@ function getDeclarationErrorAddendum(owner) {
   return '';
 }
 
+//https://github.com/facebook/react/issues/5013
+function warnIfValueIsNull(props) {
+  var valueLink = (props != null && props.valueLink != undefined) ? props.valueLink.value : '';
+
+  if (props != null && props.value === null || valueLink === null) {
```

Comment:
We probably don't need to be checking valueLink, since that is going to be deprecated in 0.15 anyway.